idea/common/src/interfaces/api-requests.ts

```diff
@@ -38,18 +38,14 @@ interface FindProgramParams extends IGenesis, Pick<IProgram, 'id'> {
   owner?: string;
 }
 
-interface AddMetaByCodeParams extends IGenesis, Pick<ICode, 'id'> {
-  metaHex: string;
+interface AddMetaParams extends IGenesis {
```

Comment:
In my opinion, it will be better to have `genesis` in the params, because we already have validation middleware that checks genesis field in the body. Also when `client` for working with `idea` definitely needs to select a network, it's not a big problem if in the request body will exists `genesis` param.

Reply:
But what if I want to get metadata by its hash without any network context?